scripts/setup/docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
do we need make.bat at all?

Reply:
(perhaps, we need to fix paths, similar to Makefile)

Reply:
> do we need make.bat at all?

I don't know as I simply used files generated as specified in Sphinx online setup docs 'sphinx-quickstart'. Perhaps not but I haven't tested this yet but I will test now